NETCORE/src/Shared/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -59,6 +59,12 @@ public static partial class ApplicationInsightsExtensions
         private const string DeveloperModeForWebSites = "APPINSIGHTS_DEVELOPER_MODE";
         private const string EndpointAddressForWebSites = "APPINSIGHTS_ENDPOINTADDRESS";
 
+        [SuppressMessage("Microsoft.Performance", "CA1823:AvoidUnusedPrivateFields", Justification = "Used in NetStandard2.0 build.")]
+        private const string ApplicationInsightsSectionFromConfig = "ApplicationInsights";
+
+        [SuppressMessage("Microsoft.Performance", "CA1823:AvoidUnusedPrivateFields", Justification = "Used in NetStandard2.0 build.")]
+        private const string TelemetryChannelSectionFromConfig = "ApplicationInsights:TelemetryChannel";
+
         [SuppressMessage("Microsoft.Performance", "CA1823:AvoidUnusedPrivateFields", Justification = "Used in NetStandard2.0 build.")]
         private const string EventSourceNameForSystemRuntime = "System.Runtime";
```

Comment:
instead of the SuppressMessage, i think you can surround these with the preprocessor `#if NetStandard`

Reply:
We had TelemetryChannelSectionFromConfig properties using this suppress message, hence followed same pattern. If including preprocessor is a better pattern, will get it changed.

Reply:
as-is, these constants will still get allocated. we've just suppressed the warning.
I recommend changing this to the preprocessor to completely remove the constants when they're not in use.
This would better align with how this file was written.

Reply:
Makes sense. I will modify to include preprocessor directive.